util/stream.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import "crypto/cipher"
+
+// ConstantStream is a cipher.Stream which always returns
+// the same value.
+type ConstantStream struct {
```

Comment:
Just put the relevant comments on the function that produces the object of the hidden type!  No need to have an unnecessary public type just in order to hang comments off of!